DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/Services/RevitToSpeckleUnitConverter.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections.Generic;
+using Speckle.Converters.Common;
+using Speckle.Core.Kits;
+using Speckle.Core.Logging;
+
+namespace Speckle.Converters.RevitShared.Services;
+
+public sealed class RevitToSpeckleUnitConverter : IHostToSpeckleUnitConverter<DB.ForgeTypeId>
+{
+  private readonly Dictionary<DB.ForgeTypeId, string> _unitMapping = new();
+
+  public RevitToSpeckleUnitConverter()
+  {
+    _unitMapping[DB.UnitTypeId.Millimeters] = Units.Millimeters;
+    _unitMapping[DB.UnitTypeId.Centimeters] = Units.Centimeters;
+    _unitMapping[DB.UnitTypeId.Meters] = Units.Meters;
+    _unitMapping[DB.UnitTypeId.Inches] = Units.Inches;
+    _unitMapping[DB.UnitTypeId.Feet] = Units.Feet;
+  }
+
+  public string ConvertOrThrow(DB.ForgeTypeId hostUnit)
```

Comment:
This should just be `Convert`? If it throws it should just be documented on the XML comments